package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build:clean": "turbo run build:clean --filter={./packages/*}",
     "build:sample-apps": "turbo run build --filter={./examples/*}",
-    "build": "turbo run build --filter={./packages/*} --concurrency=31",
+    "build": "turbo run build --filter=./packages/* --concurrency=31",
```

Comment:
Why this changed? Something is not working on your environment? Should we change the others as well?